README.md

```diff
@@ -1,568 +1,183 @@
 <div align="center"><picture>
```

Comment:
Also missing in this is supported model parsers. That's a really important section that should remain in the main readme.

Reply:
The old version had that -- that's one section that's critical to have